modules/oe_bootstrap_theme_paragraphs/oe_bootstrap_theme_paragraphs.module

```diff
@@ -329,6 +330,66 @@ function oe_bootstrap_theme_preprocess_paragraph__oe_timeline(array &$variables)
   $variables['hide_from'] = $paragraph->get('field_oe_timeline_expand')->value;
 }
 
+/**
+ * Implements hook_preprocess_paragraph() for paragraph--oe-content-row--variant-inpage-navigation.html.twig.
+ */
+function oe_bootstrap_theme_preprocess_paragraph__oe_content_row__variant_inpage_navigation(array &$variables): void {
+  $paragraph = $variables['paragraph'];
+
+  if (!$paragraph->get('field_oe_title')->isEmpty()) {
+    $variables['title'] = $paragraph->get('field_oe_title')->value;
+  }
+
+  $variables['full_layout'] = TRUE;
+  $variables['scrollspy_id'] = 'bcl-inpage-navigation';
```

Comment:
in case this id is mandatory please add a todo refering to a follow up ticket that fixes this.